modules/oe_content_persistent/oe_content_persistent.post_update.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for OpenEuropa Content Persistent module.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Update PURL configuration.
+ */
+function oe_content_persistent_post_update_00001(): void {
+  $purl_config = \Drupal::configFactory()->getEditable('oe_content_persistent.settings');
+  $purl_config->set('supported_entity_types', ['node' => 'node']);
```

Comment:
We cannot assume node. The service parameters could have been overridden so we need to load the service params and update with what the site that uses the module has set.